src/lib/server/websearch/generateQuery.ts

```diff
@@ -40,7 +40,7 @@ Current Question: What about Mexico?
 			content: `Previous questions: 
 - When is the next formula 1 grand prix?
 
-Current Question: Where is it being hosted ?`,
+Current Question: Where is it being hosted?`,
```

Comment:
because it is not 🇫🇷 : we don't need space before question mark

Reply:
damn, getting called out :laughing: good catch though!